Made some minor tweaks, mostly adding comments

neuromantic99 opened the pull request:
Also reorganised game_mechanics to make it more readble. Addded a couple tests to make sure i didn't break anything

Henry-Pulver commented on pyproject.toml:
```diff
@@ -23,12 +23,14 @@ authors = ["Your Name <you@example.com>"]
-[build-system]
-requires = ["poetry-core>=1.0.0"]
-build-backend = "poetry.core.masonry.api"
+# Commenting this out allows local pip install
+# May break replit
+# [build-system]
+# requires = ["poetry-core>=1.0.0"]
+# build-backend = "poetry.core.masonry.api"
```
Happy to go on faith that you've tested this works when both:
1. Installing to a replit
2. `pip` installing locally

neuromantic99 committed:
Made some minor tweaks, mostly adding comments

Also reorganised game_mechanics to make it more readble.
Addded a couple tests to make sure i didn't break anything
